coverage.out

```diff
@@ -0,0 +1,113 @@
+mode: count
```

Comment:
remove this file.

Reply:
Try to avoid  `git add .` command. 

make habit to add single file `git add <file name>`